src/app/containers/Figure/index.jsx

```diff
@@ -27,7 +27,7 @@ FigureContainer.propTypes = {
   src: string.isRequired,
   ratio: number.isRequired,
   copyright: string,
-  caption: string,
+  caption: node,
```

Comment:
Regardless of which way we do this, I think we should distinquish `caption` and `captionContents` as this `node` isnt the caption, its whats inside it. Not very clear imo